test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -23,6 +23,9 @@
     <PackageReference Include="Greg" Version="2.3.0.2505">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="JetBrains.dotMemoryUnit" Version="3.2.20220510">
```

Comment:
is this going to make it's way into the bin folder when deployed to artifactory? I think @pinzart90 had a way to avoid that for some test only binaries.

Reply:
I think we probably need to add to third party dep list at some point, especially if we distribute.

Reply:
No need to deploy this binary at this point. @pinzart90, do we have an example somewhere?